### PR TITLE
First draft of discrim task scoring

### PR DESCRIPTION
**Pull request by JamesOwers**
So this runs if you stick those files mono.csv and poly.csv into the the home directory.

TODO:
* Add arguments to script to allow user to specify where the files to be compared are located
* Do some checking (probably important to insist they're scoring the same primes!)
* make nice HTML and Latex output tables for the results (super easy with pandas)

### CODE REVIEW

File: evaluate_discrimination.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    nll_var = np.var(nll)
+    return nr_obs, accuracy, crossent, nll_var
+
+
+if __name__ == '__main__':
+    # TODO: Parse args
+    
+    # TODO: Get files to score
+    files = ['./mono.csv', './poly.csv']
+
```
Comment (BeritJanssen) [inline evaluate_discrimination.py]: The files should also come from the config, ideally. Locally, I'm using a separate config for mono and poly.

**Comment by BeritJanssen** [comment]
Nice! I cannot test right now. Can you organize this so that we can score all algorithms at once (poly / mono separately) and combine that in one table?

Also, it may be worth checking what format the tables on the mirex wiki have (I think it's similar to but not quite LaTeX), to see whether we can just drop those tables into the wiki. Perhaps you can open last year's result page, edit, drop in a table and preview without saving the changes?

Thanks!

**Comment by JamesOwers** [comment]
> Nice! I cannot test right now. Can you organize this so that we can score all algorithms at once (poly / mono separately) and combine that in one table?

Of course, should be fine.

> Also, it may be worth checking what format the tables on the mirex wiki have (I think it's similar to but not quite LaTeX), to see whether we can just drop those tables into the wiki. Perhaps you can open last year's result page, edit, drop in a table and preview without saving the changes?

Will give that a shot.

**Comment by JamesOwers** [comment]
Right oh, I think that's all done. I tested pasting the .html files directly in the wiki, and they render just fine. The output is a single file - `discrim_table.html`. Additions required to `config.py` updated in the README.md.

**Comment by BeritJanssen** [comment]
@JamesOwers , I finally managed to test now. I assumed the `DISCRIM_FILES` variables were supposed to contain dictionaries with the model name as key and the file path to the .csv as the value. I get `AttributeError: 'float' object has no attribute 'exp'`. Do you have a hunch where this comes from?

**Comment by BeritJanssen** [comment]
Ah, never mind. My mistake. I grabbed last year's submissions, but these were calculated from an older version of the dataset, with different file names.

**Pull request merged**